config/default.go

```diff
@@ -44,6 +44,9 @@ test:
 record:
   recordTimer: 0s
   filters: []
+contract:
```

Comment:
You don't need to set these default values, it should be empty, better to just add an example here

Reply:
Okay, remove this once it is completely tested.